test/helpers/cilium.go

```diff
@@ -333,6 +349,96 @@ func (c *Cilium) ReportFailed(commands ...string) {
 		fmt.Fprint(wr, res.Output())
 	}
 	fmt.Fprint(wr, "StackTrace Ends\n")
+	c.ReportFailedFull()
+}
+
+//ReportFailedFull runs a variety of commands and write the results to
+//testResultsPath
+func (c *Cilium) ReportFailedFull() {
```

Comment:
I'm not a fan of this function name. What about CiliumReportDump? A dump is a widely used term, while "Full" is somewhat murky.